static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js

```diff
@@ -4,8 +4,7 @@ define([
     'common/utils/$',
     'common/utils/mediator',
     'common/utils/storage',
-    'common/utils/template',
```

Comment:
I wish we could, but `template` is used in `sport.js` where the string is dynamically assigned